tests/other_test.go

```diff
@@ -104,3 +104,24 @@ func TestPlanICDVersions(t *testing.T) {
 		t.Run(version, func(t *testing.T) { testPlanICDVersions(t, version) })
 	}
 }
+
+func TestRunBasicExampleWithFlavorMultitenant(t *testing.T) {
```

Comment:
same comment as below, is there an existing test we can use to test `multitenant` ? We might even want to use one of the ones in pr_test.go so that use case is tested for every PR to check for regressions

Reply:
Refactored to include multitenant in complete example and removed this test in latest 3414351